cassandra-analytics-integration-tests/src/test/java/org/apache/cassandra/analytics/SharedClusterSparkIntegrationTestBase.java

```diff
@@ -84,7 +84,7 @@ protected DataFrameReader bulkReaderDataFrame(QualifiedName tableName)
                   // Shutdown hooks are called after the job ends, and in the case of integration tests
                   // the sidecar is already shut down before this. Since the cluster will be torn
                   // down anyway, the integration job skips clearing snapshots.
-                  .option("clearSnapshot", "false")
+                  .option("clearSnapshotStrategy", "OnCompletion")
```

Comment:
since `onCompletion` behaviour was similar to test setup (clear after test method completes), kept that. But for consistency, will change it to `noop`